java/tasks/main.yml

```diff
@@ -3,13 +3,7 @@
 -
   include: linux.yml
   when: ansible_distribution != 'MacOSX'
-  tags:
```

Comment:
Potentially change this to `ansible_distribution == 'CentOS ' or ansible_distribution == 'Red Hat Enterprise Linux'` if its not working on Fedora

Reply:
@PhilipGough yes will do that. I've also added a note to this JIRA [1] as suggested by @aliok but I think it's possible to make this change quickly.

[1] https://issues.jboss.org/browse/AGDIGGER-216